app/views/brexit_checker/_results_citizen_actions.html.erb

```diff
@@ -0,0 +1,31 @@
+<% heading = t('brexit_checker.results.audiences.citizen.heading') %>
+<section
+  class="brexit-checker-actions"
+  data-analytics-ecommerce
+  data-ecommerce-start-index="1"
+  data-list-title="Brexit checker results: <%= heading %>"
```

Comment:
This heading for the ecommerce tracking needs to be the sub-group heading, rather than the main parent group heading. E.g: "Visiting the EU" rather than "You and your family"